images/framework/cos.yaml

```diff
@@ -0,0 +1,21 @@
+repositories:
+- &cos
+  name: "cos"
+  description: "cOS official"
+  # Repository type
+  type: "docker"
+  # Repository cache
+  cached: true
+  # Repository priority
+  priority: 1
+  # Repository architecture
+  arch: "amd64"
+  # Enable/disable Docker notary checks
+  # 
+  verify: false
+  urls:
+  - "quay.io/costoolkit/releases-green"
+- <<: *cos
+  arch: "arm64"
```

Comment:
umm, AFAIK luet is (was?) arch agnostic? How would this work? Inst this setting boths repos for both arches?

Reply:
so luet _is_ arch agnostic, but allows to have clients define arch for repositories, in which case gets automatically enabled if the relevant arch matches in runtime.

https://rancher-sandbox.github.io/cos-toolkit-docs/docs/examples/creating_bootable_images/#configuration-file